bin/brakeman

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
```

Comment:
besides these lines, verbatim copy of https://github.com/rails/rails/blob/v8.0.4/railties/lib/rails/generators/rails/app/templates/bin/brakeman.tt